docs/application/native/guides/app-management/sticker.md

```diff
@@ -0,0 +1,427 @@
+# Sticker
+
+
+The Sticker feature provides users with fun experiences to share sticker information. The Sticker applications are composed of a [Sticker provider application](#provider_application) and [Sticker consumer application](#consumer_application). The Sticker provider creates stickers and developers can create their own Sticker consumer applications that use the stickers created by the Sticker provider. The Sticker provider application can store sticker details such as URI, keyword, and group name to the Sticker database. The Sticker consumer applications can retrieve sticker information using the Sticker API.
+
+The main features of the Sticker API include:
+
+- Preparing the Sticker service for use
+
+  You can connect the Sticker daemon to operate the Sticker service.
+
+- Sticker data handle
+
+  You can use the Sticker data handle to [store the sticker information](#set_data_handle) and to [get the sticker information](#get_data_handle) stored in it.
+
+- [Inserting the sticker information](#insert_sticker)
+
+  - You can store the sticker information using the Sticker API at application runtime, and [use the JSON file](#set_json) to store the sticker information at a time.
+  - You can store sticker information automatically when the Sticker provider application is installed without using the Sticker API.
+
+- Retrieving the sticker information
+
+  You can [retrieve sticker information](#retrieve_sticker), which includes URI, keyword, group name, and so on.
```

Comment:
Please add article "the" before sticker information. Reference comment: https://github.com/Samsung/tizen-docs/pull/884#discussion_r336347760

You can [retrieve the sticker information](#retrieve_sticker), which includes URI, keyword, group name, and so on.

Reply:
Fixed.